.github/workflows/test-pytorch-xla-tpu.yml

```diff
@@ -31,4 +31,4 @@ jobs:
 
       - name: Build and test optimum tpu
         run: |
-          HF_TOKEN=${{ secrets.HF_TOKEN_OPTIMUM_TPU_CI }} make tests
+          JETSTREAM_PT_DISABLE=1 HF_TOKEN=${{ secrets.HF_TOKEN_OPTIMUM_TPU_CI }} make tests
```

Comment:
This should be on all test-pytorch-xla-* tests no?

https://github.com/huggingface/optimum-tpu/tree/main/.github/workflows